broker-plugins/broker-query-engine/src/main/java/org/apache/qpid/server/query/engine/evaluator/settings/QuerySettings.java

```diff
@@ -78,7 +77,7 @@
     /**
      * ZoneId used in date/time representation
      */
-    private ZoneId _zoneId = ZoneId.of(HttpPort.DEFAULT_QUERY_ENGINE_ZONE_ID);
+    private ZoneId _zoneId = ZoneId.of("UTC");
```

Comment:
The default value constant could be defined in this class (or in some other class in query engine plugin) and reused in HttpManagementConfiguration to avoid duplication of default values.